osu.Game.Modes.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -95,14 +102,22 @@ protected override void UpdateInitialState()
         {
             base.UpdateInitialState();
             body.Alpha = 1;
+
+            //we need to be visible to handle input events. note that we still don't get enough events (we don't get a position if the mouse hasn't moved since the slider appeared).
+            ball.Alpha = 0.01f; 

```

Comment:
I'm merging this in under the assumption it'll be addressed once we pass input states to Update.